src/__experimental__/components/simple-color-picker/simple-color-picker.component.js

```diff
@@ -1,19 +1,131 @@
-import React, { useState, useRef, useEffect } from 'react';
+import React, {
+  useCallback,
+  useState,
+  useRef,
+  useEffect
+} from 'react';
 import PropTypes from 'prop-types';
-
+import Events from '../../../utils/helpers/events';
 import tagComponent from '../../../utils/helpers/tags/tags';
 import SimpleColor from './simple-color';
 import RadioButtonMapper from '../radio-button/radio-button-mapper.component';
 import { SimpleColorFieldset, StyledColorOptions } from './simple-color-picker.style';
 
 const SimpleColorPicker = (props) => {
   const {
-    children, name, legend, onChange, onBlur, value, isBlurBlocked = false
+    children,
+    name,
+    legend,
+    onChange,
+    onBlur,
+    onKeyDown,
+    value,
+    isBlurBlocked = false,
+    maxWidth = 300,
+    childWith = 58
   } = props;
 
   const myRef = useRef(null);
   const [blurBlocked, setIsBlurBlocked] = useState(isBlurBlocked);
   const [focusedElement, setFocusedElement] = useState(null);
+  const totalLength = childWith * children.length;
+  const itemsPerRow = Math.floor(maxWidth / childWith);
+  const rowCount = Math.ceil(children.length / itemsPerRow);
+  const remainder = (((itemsPerRow * rowCount) * childWith) - totalLength) / childWith;
```

Comment:
I think that could be just :
const remainder = itemsPerRow * rowCount - children.length;

Reply:
yes, fixed: 8a84ed7